tools.md

```diff
@@ -0,0 +1,90 @@
+
+# Communication Tools
+
+Here is a quick introduction to the common, and free, communication tools that 18F teams use when working with other federal agencies. 18F can help you decide which tools best fit your needs and help you get set up.
```

Comment:
Although these tools are freely available and have free tiers the pricing model of these tools that we're on is usually a paid version the features we need for security and compliance aren't typically offered in the free tier.

Reply:
@afeld & @JJediny -- these tools should probably be considered 'widely-used' enough to merit sponsorship through FedRAMP

Reply:
I suspect we should probably close this in favor of the efforts currently ongoing in the handbook.

cc @colinmurphy01

Reply:
Agreed and we'll be posting a new PR today.